### PR TITLE
Migrate off deprecated window API

### PR DESCRIPTION
**Pull request by goderbauer**
Fixes https://github.com/flutter/gallery/issues/923.

### CODE REVIEW

File: test_goldens/testing/util.dart [hunk 1]
```diff
@@ -114,10 +114,11 @@ Future<void> setUpBinding(
...
   Brightness brightness = Brightness.light,
 }) async {
-  tester.binding.window.physicalSizeTestValue = size;
-  tester.binding.window.devicePixelRatioTestValue = 1.0;
-  tester.binding.window.platformDispatcher.textScaleFactorTestValue = 1.0;
-  tester.binding.window.platformDispatcher.platformBrightnessTestValue =
+  tester.view.physicalSize = size;
+  tester.view.devicePixelRatio = 1.0;
+  tester.view.platformDispatcher.textScaleFactorTestValue = 1.0;
+  tester.view.platformDispatcher.platformBrightnessTestValue =
```
Comment (pdblasi-google) [inline test_goldens/testing/util.dart:120]: There should be a `tester.platformDispatcher` property to use for these.

File: test_goldens/testing/util.dart [hunk 2]
```diff
@@ -114,10 +114,11 @@ Future<void> setUpBinding(
...
       brightness;
-  addTearDown(tester.binding.window.clearAllTestValues);
+  addTearDown(tester.view.reset);
+  addTearDown(tester.view.platformDispatcher.clearAllTestValues);
```
Comment (pdblasi-google) [inline test_goldens/testing/util.dart:123]: Same here, you should be able to just use `tester.platformDispatcher`.

File: test_goldens/testing/util.dart (hunk 1 above)
Comment (goderbauer) [reply test_goldens/testing/util.dart]: Done.

File: test_goldens/testing/util.dart (hunk 2 above)
Comment (goderbauer) [reply test_goldens/testing/util.dart]: Done.

**Review by pdblasi-google — APPROVED** [review]

**Pull request merged**